lib/advisor/core/answers/answers.ex

```diff
@@ -1,13 +1,30 @@
-defmodule Advisor.Core.AnswerFinder do
-  alias Advisor.Core.Answer
+defmodule Advisor.Core.Answers do
```

Comment:
A small naming convention issue maybe: This seems similar to the `AdviceFinder` in terms of functionality but it's not called `Finder`. The same applies for entities below.